internal/app/minimatch/minimatch.go

```diff
@@ -51,9 +56,39 @@ func RunApplication() {
 		}).Fatalf("cannot bind server.")
 	}
 
+	if err := newMinimatchCmd(cfg).Execute(); err != nil {
+		minimatchLogger.Fatal(err)
+		return
+	}
+
 	rpc.MustServeForever(p)
 }
 
+func newMinimatchCmd(cfg config.View) *cobra.Command {
```

Comment:
I thought we decided to make this a separate tool. It's possible to shell exec and communicate that way.

Reply:
+1 - lets discuss this in person so we can build a more reusable client load gen if required.